pages/events/fall-fest.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <main class="fall-fest-page">
+  <div class="fall-fest-page">
```

Comment:
Same, `article`?

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/3441/commits/8813034834a0b3ef57c06bdf5a74df881100161f